engine/src/test/java/io/camunda/zeebe/engine/processing/timer/TimerStartEventTest.java

```diff
@@ -1192,4 +1199,43 @@ public void shouldAvoidTimeShifting() {
                 .count())
         .isEqualTo(2);
   }
+
+  @Test
+  public void shouldCreateCronTimer() {
+    // when
+    final var now = ZonedDateTime.now(ZoneId.systemDefault()).withSecond(0).withNano(0);
+    engine.getClock().setCurrentTime(now.toInstant());
+
+    final var dueDate = now.plusSeconds(10).toInstant().toEpochMilli();
```

Comment:
🔧 We should make this test less dependent on execution speed. Currently this test looks prone to flakiness (although, please correct me if I'm wrong)

My current understanding is that the engine's clock might move along a few seconds (if the test is being executed slowly, for example on a machine that is busy running other tests), before the engine is able to process the deploy command that creates the timer with due date. When determining the due date (CRON schedule + current time), the current time may already have moved along a bit compared to the expected due date.

❓ What do you think? Is that thinking correct? Can we find a way to test this without being dependent on an exact second?